data/language/dutch.txt

```diff
@@ -1709,7 +1709,7 @@ STR_1707    :Teveel werknemers in dit park
 STR_1708    :{SMALLFONT}{BLACK}Werkgebied van deze werknemer instellen
 STR_1709    :Werknemer ontslaan
 STR_1710    :Ja
-STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wilt ontslaan?
+STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wil ontslaan?
```

Comment:
Informeel is geen zwart-witkwestie. Het spel gebruikt informele taal, anders werd de speler niet met 'jij' aangesproken. De discussie ging over _hoe_ informeel. En als bij woordkeuzes als 'aanleggen' en 'doelstelling' past 'jij wilt' beter dan het veel informelere 'jij wil'.